pandas-stubs/io/formats/style.pyi

```diff
@@ -1,97 +1,333 @@
+        axis: None,
+        subset: Subset | None = ...,
+        **kwargs: Any,
+    ) -> Styler: ...
+    def apply_index(
+        self,
+        func: Callable[[Series], npt.NDArray[np.str_] | list[str] | Series[str]],
+        axis: int | str = ...,
+        level: Level | list[Level] | None = ...,
+        **kwargs: Any,
+    ) -> Styler: ...
+    def applymap_index(
+        self,
+        func: Callable[[object], str],
+        axis: int | str = ...,
+        level: Level | list[Level] | None = ...,
+        **kwargs: Any,
+    ) -> Styler: ...
+    def applymap(
+        self, func: Callable[[object], str], subset: Subset | None = ..., **kwargs: Any
+    ) -> Styler: ...
+    def set_table_attributes(self, attributes: str) -> Styler: ...
+    def export(self) -> StyleExportDict: ...
+    def use(self, styles: StyleExportDict) -> Styler: ...
+    uuid: Any  # Incomplete
+    def set_uuid(self, uuid: str) -> Styler: ...
+    caption: Any  # Incomplete
+    def set_caption(self, caption: str | tuple[str, str]) -> Styler: ...
+    def set_sticky(
+        self,
+        axis: Axis = ...,
+        pixel_size: int | None = ...,
+        levels: Level | list[Level] | None = ...,
+    ) -> Styler: ...
+    def set_table_styles(
+        self,
+        table_styles: dict[HashableT, CSSStyles] | CSSStyles | None = ...,
+        axis: int = ...,
+        overwrite: bool = ...,
+        css_class_names: dict[str, str] | None = ...,
+    ) -> Styler: ...
+    def hide(
+        self,
+        subset: Subset | None = ...,
+        axis: Axis = ...,
+        level: Level | list[Level] | None = ...,
+        names: bool = ...,
+    ) -> Styler: ...
     def background_gradient(
         self,
-        cmap=...,
-        low=...,
-        high=...,
-        axis=...,
-        subset=...,
-        text_color_threshold=...,
+        cmap: str | Colormap = ...,
+        low: float = ...,
+        high: float = ...,
+        axis: Axis | None = ...,
+        subset: Subset | None = ...,
+        text_color_threshold: float = ...,
+        vmin: float | None = ...,
+        vmax: float | None = ...,
+        gmap: Sequence[float]
+        | Sequence[Sequence[float]]
+        | npt.NDArray
+        | DataFrame
+        | Series
+        | None = ...,
+    ) -> Styler: ...
+    def text_gradient(
+        self,
+        cmap: str | Colormap = ...,
+        low: float = ...,
+        high: float = ...,
+        axis: Axis | None = ...,
+        subset: Subset | None = ...,
+        # In docs but not in function declaration
+        # text_color_threshold: float
         vmin: float | None = ...,
         vmax: float | None = ...,
-    ): ...
-    def set_properties(self, subset=..., **kwargs): ...
+        gmap: Sequence[float]
+        | Sequence[Sequence[float]]
+        | npt.NDArray
+        | DataFrame
+        | Series
+        | None = ...,
+    ) -> Styler: ...
+    def set_properties(
+        self, subset: Subset | None = ..., **kwargs: str | int
```

Comment:
I think `**kwargs` should be `Any` here

Reply:
I think the properties that are set must be int or str.  These are CSS properties for `<td>` cells.

Reply:
Interesting.  Didn't know you could do that with `kwargs`.  Probably means we could tighten up some other places in the stubs where `kwargs` is used.